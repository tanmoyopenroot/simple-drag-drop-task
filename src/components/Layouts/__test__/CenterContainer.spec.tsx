import * as React from 'react';
import { shallow } from 'enzyme';

import {
  CenterContainer,
  ICenterContainerProps,
} from '../index';

describe('Testing <CenterContainer />', () => {
  const props: ICenterContainerProps = {
    width: '100%',
    color: 'gray',
  };
  const wrapper = shallow<ICenterContainerProps>(
    <CenterContainer {...props}>Container</CenterContainer>,
  );

  it('should be able to render with props', () => {
    expect(wrapper.length).toEqual(1);
    expect(wrapper.children().length).toBe(1);
    expect(wrapper.props().width).toEqual(props.width);
    expect(wrapper.props().color).toEqual(props.color);
    expect(wrapper).toMatchSnapshot();
  });
});
