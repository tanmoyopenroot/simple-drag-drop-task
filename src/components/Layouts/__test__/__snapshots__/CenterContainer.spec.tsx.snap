// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <CenterContainer /> should be able to render with props 1`] = `"<div width=\\"100%\\" color=\\"gray\\" class=\\"sc-bdVaJa cVOEHC\\">Container</div>"`;
